### DISCUSSION TITLE
Rejected from GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by nps32** [community/community#109619] [GitHub Education]
### Hi everyone,

I applied a few weeks ago for the GitHub Student Developer Package and was rejected. 

In the rejection email, it asked me to fill out my billing info and 2FA and try again. I have done so, but it continues to reject me claiming that my proof of current enrollment has already been used. I uploaded my official transcript from Georgetown University, which is not an acceptance letter but actually provides proof of my enrollment, and I am not sure if a different document would suffice. 

My Github account is associated with my school email additionally. 

I tried to submit a GitHub Customer Service ticket but it was disabled for issues of this category. I would like to use copilot for my personal projects so I was wondering if anyone from GitHub Support could resolve this issue.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket :

![github04](https://github.com/community/community/assets/72712095/096b3477-cee6-4a19-9f1b-6fb2219b51a1)

and open a New Ticket:

![github05](https://github.com/community/community/assets/72712095/7413fc55-875a-498c-93eb-5c54d5ffc8d6)


God bless you!

**Comment by TioSatrio100** [comment]
I've also experienced this. I resolved it by sending different academic transcript photo

**Comment by UpekHansaja** [comment]
Hi there,

It seems like your GitHub profile is not as they requested,

### Set Up 2FA

1. Go to [Security section of Settings page](https://github.com/settings/security) of your Github account.
2. Scroll down a bit and look for Two-factor authentication and in there, Setup 'Authenticator app' from it (Sometimes it's better to use Github Mobile App on your phone). 

![image](https://github.com/community/community/assets/109482062/3f144249-f8f0-416c-843b-427d3996cd3b)

Make sure you got same as the above image.
after you done with 2FA, lets' heading to lets config billing section

### Billing Info

1. Go to [Payment Information at the Billings and Plans section of Settings page](https://github.com/settings/billing/payment_information) of your Github account.
2. In there complete your billing informations such as billing address, payment method (Credit/Debit Card details), etc..
3. Make sure to check those informations are true.

don't worry no payment will credited from you acc.
after you done with billing informations, You have to create your arrange your public profile.

### Setting up public profile

This is the **Secret** to obtain [Github Student Developer Pack](https://education.github.com/pack)
1. Having a Public Profile Picture - This helps to understand that you have an identity. (Always make sure to use a real photo of your self across the social media platforms like linkedIn, Facebook, etc.. and use the same to on GitHub platform as well).
2. Go to your [public profile](https://github.com/settings/profile) section of your GitHub settings page, and complete the public profile. (In this case make sure to complete your public profile with the true details of your self as you provided for your college or campus)
3. Having a Bio - add bio to your public profile, so it can help to understand what you are currently doing by looking active (in here mention you are a student who is studying regarding the industry such as Undergraduate in Software Engineering, Undergraduate in Computer Science, Undergraduate : BSc(hons) in computing etc...)
4. Link your Social Media - Add some of your social media links such as, LinkedIn, Twitter, Facebook, Instagram, etc. to Social accounts section of your public profile.
5. Finally provide your current location (country) which your collage/uni and you are living along with the timezone.

by following above steps, make sure to always save or update your informations before closing the browser.

Finally you have to create a repo from your username and it help to make your profile unique to yourself...

If you having trouble to understand the above, and incase if your need a reference kindly go though with [my GitHub profile](https://github.com/UpekHansaja), hope this helps you..

Thanks & Regards...
